packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -79,7 +88,35 @@ describe('quantic-generated-answer', () => {
           Expect.logDislikeGeneratedAnswer(streamId);
           Expect.likeButtonIsChecked(false);
           Expect.dislikeButtonIsChecked(true);
+          Expect.displayFeedbackModal(true);
+        });
+
+        scope('when closing the feedback modal', () => {
+          Actions.clickFeedbackCancelButton();
+          Expect.displayFeedbackModal(false);
         });
+
+        scope('when selecting a feedback option', () => {
```

Comment:
following my other question, should we test that a feedback is submitted if no value/detail is given by the user?

Reply:
Actually we already have a unit that covers that included in the unit test of the Quantic Feedback Modal component: https://github.com/coveo/ui-kit/blob/8afa4e415ecdb60dc47c0b4e21a48302c7311976/packages/quantic/force-app/main/default/lwc/quanticFeedbackModal/__tests__/quanticFeedbackModal.test.js#L279